core/controlplane/config/templates/cloud-config-worker

```diff
@@ -634,9 +634,9 @@ write_files:
              mv -f $f ${encKey%.enc}
            done;
 
-           {{ if .Experimental.TLSBootstrap.Enabled }}
+           {{ if and .Experimental.TLSBootstrap.Enabled .AssetsConfig.HasTLSBootstrapToken }}
            echo injecting token into the kubelet bootstrap kubeconfig file
-           bootstrap_token=$(cat /etc/kubernetes/auth/kubelet-bootstrap.token);
+           bootstrap_token=$(cat /etc/kubernetes/auth/tls-bootstrap.token);
```

Comment:
Yes, my reasoning was to make it match with the symbols in Go code, and what this feature is called in upstream [Kubernetes documentation](https://kubernetes.io/docs/admin/kubelet-tls-bootstrapping/), but I think I forgot to replace a few other names in templates.

Do you think it's the right thing to do here? Keep the old `kublet-bootstrap` names throughout the templates, or replace them all by `tls-bootstrap` (or even `kubelet-tls-bootstrap`)?